yew/src/html/classes.rs

```diff
@@ -0,0 +1,242 @@
+use indexmap::IndexSet;
+use std::{
+    borrow::{Borrow, Cow},
+    iter::FromIterator,
+};
+
+/// A set of classes. The use of `classes!` is preferred.
```

Comment:
```suggestion
/// A set of classes.
/// 
/// Using [`classes!`][yew::classes!] for initialization is preferred.
```

I like to draw parallels between `Vec/vec!` and `Classes/classes!` and well... Let's just say our documentation doesn't hold up in comparison :smile:
Well anyway, I'm hoping some day a magic documentation guru shows up and writes the perfect documentation for us :pray: